lib/metasploit/framework/ntds/account.rb

```diff
@@ -0,0 +1,178 @@
+module Metasploit
+  module Framework
+    module NTDS
+      # This class represents an NTDS account structure as sent back by Meterpreter's
+      # priv extension.
+      class Account
+
+        # @!attribute description
```

Comment:
YARD has advised in an issue on their repository that `@!attribute` should not be used with `attr_accessor`.  You can still use a comment and @return, there just should be no `@!attribute` and the text shouldn't be indented